src/xdocs/config_annotation.xml

```diff
@@ -1385,23 +1443,37 @@ private static final int m = 0;
              the <code>aliasList</code>:
         </p>
         <source>
-@SuppressWarnings("paramnum")
-public void needsLotsOfParameters(@SuppressWarnings("unused") int a,
-  int b, int c, int d, int e, int f, int g, int h) {
-  ...
+&lt;module name=&quot;TreeWalker&quot;&gt;
+  &lt;module name=&quot;ParameterNumber&quot;/&gt;
+
+  &lt;module name=&quot;SuppressWarningsHolder&quot;&gt;
+    &lt;property name=&quot;aliasList&quot; value=
+      &quot;com.puppycrawl.tools.checkstyle.checks.sizes.ParameterNumberCheck=paramnum&quot;/&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+&lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+        </source>
+        <source>
+class Test {
+
+   public void needsLotsOfParameters int a, // violation
+      int b, int c, int d, int e, int f, int g, int h) {
+      // ...
+   }
+
+   @SuppressWarnings("paramnum")
+   public void needsLotsOfParameters1 int a, // violation suppressed
+      int b, int c, int d, int e, int f, int g, int h) {
+      // ...
+   }
+
 }
         </source>
 
         <p>
           It is possible to suppress all the checkstyle warnings with the argument
           <code>"all"</code>:
```

Comment:
Please move this sentence to description, to be after `using name of the check to be excluded.`

Reply:
done